python/votingapp/index.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, render_template, request, make_response, g
+import os
+import socket
+import random
+import json
+import requests
+
+option_a = os.getenv('OPTION_A', "Mountains")
+option_b = os.getenv('OPTION_B', "Beaches")
+hostname = socket.gethostname()
+
+app = Flask(__name__)
+
```

Comment:
I think we can fit worker and this code within a single function and use it serve dashboard and as well as cast vote.

We can try with prefix-based path if we can get it working.
Or host this server as a container function.

Essentially idea should be all components of our app are hosted on Fission.

Reply:
Done.